react-native-in-app-payments-quickstart/app/components/DigitalWalletButton.tsx

```diff
@@ -29,7 +29,7 @@ DigitalWalletButton.propTypes = {
   onPress: PropTypes.func.isRequired,
 };
 
-export default function DigitalWalletButton({ onPress }) {
+export default function DigitalWalletButton({ onPress }: { onPress: any }) {
```

Comment:
please use a concrete type instead of any

Reply:
Resolved it.